src/components/TodoList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+import { TodoInfo } from './TodoInfo';
+
+type Props = {
+  todos: Todo[],
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <ul className="todoapp__main">
+      {
+        todos.map(todo => {
+          return <TodoInfo key={todo.id} todo={todo} />;
+        })
+      }
```

Comment:
it can be simplified
```suggestion
      {todos.map(todo => (
        <TodoInfo key={todo.id} todo={todo} />
      )}
```